noty-android/app/composeapp/src/main/java/dev/shreyaspatil/noty/composeapp/view/screen/NotesScreen.kt

```diff
@@ -73,6 +74,11 @@ fun NotesScreen(
                 elevation = 0.dp,
                 actions = {
                     ThemeSwitchAction(toggleTheme)
+                    AboutAction {
+                        lifecycleScope.launch {
+                            navigateToAbout(navController)
+                        }
+                    }
```

Comment:
And I think there's no need to have a separate function since we're using it once. So make it direct.

```kotlin
AboutAction {
    navController.navigate(Screen.About.route)
}
```

Reply:
Here I followed the separation of concerns (UI and BusinessLogin)